Sources/CardPayments/Models/CardRequest.swift

```diff
@@ -11,14 +11,24 @@ public struct CardRequest {
     /// 3DS authentication launch option
     public let sca: SCA
     
+    /// The option to vault with purchase
```

Comment:
What does "vault with purchase" mean? Can we be a bit more descriptive?

Reply:
Addressed in commit 6057b4e6155b57c8c2fa4f361c0410eb0f29f1e3